commit 95511a2a92ab95f75734a7460325887daa93cbcd (refs/changes/52/3252/2)
Author: cjihrig <cjihrig@gmail.com>
Date:   2018-01-23T13:10:41-05:00 (1 year, 8 months ago)
    
    SSO-202 Upgrade sdcsso to Node 8 (sdcsso zone setup requires postfix)
    
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index 2ac3cb4..44d2944 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1374,7 +1374,8 @@ cat <<EOF
     "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
     "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "openssl-1.0.2k"
+      "openssl-1.0.2k",
+      "postfix-3.0.2nb2"
     ],
     "repos": [
         {"url":  "git@github.com:joyent/sdcsso.git" }
